packages/pn-pa-webapp/src/pages/NotificationDetail.page.tsx

```diff
@@ -51,10 +51,10 @@ const NotificationDetail = () => {
   );
   const detailTableRows: Array<NotificationDetailTableRow> = [
     { id: 1, label: 'Data', value: <Box fontWeight={600}>{notification.sentAt}</Box> },
-    { id: 2, label: 'Termini di pagamento', value: `Entro il ` },
+    { id: 2, label: 'Da pagare entro il', value: <Box fontWeight={600}>{notification.sentAt}</Box> },
```

Comment:
Non mi torna che il valore sia sentAt. Probabilmente non abbiamo quel valore oppure va calcolato in base a delle condizioni.
Chiedere in stand up

Reply:
questo commento è stato risolto nella 1605 da @maurizioflauti